bin/node/runtime/src/voter_bags.rs

```diff
@@ -233,3 +233,206 @@ pub const THRESHOLDS: [u64; 200] = [
 	17_356_326_621_502_140_416,
 	18_446_744_073_709_551_615,
 ];
+
+pub const THRESHOLDS_BALANCE: [crate::Balance; 200] = [
+	100_000_000_000_000,
+	106_282_535_907_434,
+	112_959_774_389_150,
+	120_056_512_776_105,
+	127_599_106_300_477,
```

Comment:
This file was auto-generated, and now you have copy pasted some duplicate code here. you should generate a new file or make this in a new file or whatever, but not modify an auto-generated file.